tests/qam-minimal/install_patterns.pm

```diff
@@ -25,14 +25,27 @@ use utils;
 use qam;
 use testapi;
 
+sub install_packages {
+    my $patch_info = shift;
+    my $pattern    = qr/\s+(.+)(?!\.src)\..*\s<\s.*/;
+
+    # loop over packages in patchinfo and try instalation
+    foreach my $line (split(/\n/, $patch_info)) {
+        if (my ($package) = $line =~ $pattern) {
+            script_run("zypper -n in $package", 700);
+            save_screenshot;
+        }
+    }
+}
+
 sub run {
     my ($self) = @_;
+    my $patch = get_var('INCIDENT_PATCH');
```

Comment:
In general, keep the case in mind that the variable is undefined. `get_var` can also take an optional default value, e.g. `get_var('PATCH', 'foo')`. If you rely on the variable being defined, use `get_required_var()`.